hardware/setup/snap_build.tcl

```diff
@@ -162,18 +167,30 @@ if { $bram_used == "TRUE" } {
     set RAM_TYPE noSDRAM
 }
 append IMAGE_NAME [format {_%s_%s_%s} $RAM_TYPE $fpgacard $TIMING_TNS]
- 
+append IMAGE_NAME [expr {$golden_image == "TRUE" ? "_FACTORY" : ""}] 
+
 ## 
 ## writing bitstream
 set step     write_bitstream
 set logfile  $log_dir/${step}.log
 set command  "write_bitstream -force -file ./Images/$IMAGE_NAME"  
 puts [format "%-*s %-*s %-*s %-*s"  $widthCol1 "" $widthCol2 "generating bitstreams" $widthCol3 "" $widthCol4 "[clock format [clock seconds] -format %H:%M:%S]"]
 
+set_property BITSTREAM.GENERAL.COMPRESS {TRUE} [ current_design ]
+set_property BITSTREAM.CONFIG.EXTMASTERCCLK_EN {DIV-4} [current_design]
+set_property CONFIG_MODE {BPI16} [current_design]
+set_property BITSTREAM.CONFIG.BPI_1ST_READ_CYCLE 4 [current_design]
+set_property BITSTREAM.CONFIG.BPI_PAGE_SIZE 8 [current_design]
```

Comment:
set_property should be part of a xdc file, otherwise it is not usable in other flows